dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -1566,6 +1566,14 @@ UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W=CNTR0341W: Unable to bind the {0} interf
 UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.explanation=The remote enterprise bean binding cannot be created because the CORBA name server is not available.
 UNABLE_TO_BIND_REMOTE_BINDING_CNTR0341W.useraction=Check the log file for exceptions that are related to the ORB or CORBA name server initialization.
 
+CNTR0342E_INCORRECT_ASYNC_ANNO=CNTR0342E: The {0} bean in the {1} module of the {2} application has one or more methods with the jakarta.enterprise.concurrent.Asynchronous annotation, which must not be used on enterprise beans. The jakarta.ejb.Asynchronous annotation provides asynchronous methods for enterprise beans.
+CNTR0342E_INCORRECT_ASYNC_ANNO.explanation=The Jakarta Concurrency specification does not allow the jakarta.enterprise.concurrent.Asynchronous annotation to be used on Jakarta Enterprise Beans.
+CNTR0342E_INCORRECT_ASYNC_ANNO.useraction=Remove the jakarta.enterprise.concurrent.Asynchronous annotation from the method or Class. Use the jakarta.ejb.Asynchronous annotation to identify asynchronous enterprise bean methods.
+
+CNTR0343W_INCORRECT_ASYNC_ANNO_INTERFACE=CNTR0343W: The {0} business interface contains a jakarta.enterprise.concurrent.Asynchronous annotation. Asynchronous annotations are not valid on any interface, and the Jakarta Enterprise Beans container will ignore them.
+CNTR0343W_INCORRECT_ASYNC_ANNO_INTERFACE.explanation=The jakarta.enterprise.concurrent.Asynchronous annotation must not be used with Jakarta Enterprise Beans. The EJB container ignores these annotations on interfaces. Unless the jakarta.ejb.Asynchronous annotation is specified on the bean class or a bean class method or the methods are declared asynchronous in the ejb-jar.xml deployment descriptor, the bean methods will run synchronously.
```

Comment:
```suggestion
CNTR0343W_INCORRECT_ASYNC_ANNO_INTERFACE.explanation=The jakarta.enterprise.concurrent.Asynchronous annotation must not be used with Jakarta Enterprise Beans. The EJB container ignores these annotations on interfaces. Unless the methods are declared asynchronous in the ejb-jar.xml deployment descriptor or the jakarta.ejb.Asynchronous annotation is specified on the bean class or a bean class method, the bean methods run synchronously.
```

Reply:
Please do not use "EJB" in any new messages.  "EJB" is a registered trademark and is no longer used to refer to the Jakarta Enterprise Beans technology.  Use either "Jakarta Enterprise Beans conainer" or "enterprise beans container".  It is of course acceptable to use "ejb" when it is part of the file name (ejb-jar.xml) or API package (jakarta.ejb.Asynchronous).